src/download.ts

```diff
@@ -68,6 +68,28 @@ export class MinikubeDownload {
     return latestReleases[0];
   }
 
+  async selectVersion(cliInfo?: extensionApi.CliTool): Promise<MinikubeGithubReleaseArtifactMetadata> {
+    let lastReleasesMetadata = await this.grabLatestsReleasesMetadata();
+
+    if (lastReleasesMetadata.length === 0) throw new Error('cannot grab minikube releases');
+
+    // if the user already has an installed version, we remove it from the list
+    if (cliInfo) {
+      lastReleasesMetadata = lastReleasesMetadata.filter(release => release.tag.slice(1) !== cliInfo.version);
+    }
```

Comment:
does it work with cliInfo?.version (no usage of the if)

Reply:
Humm, yes but we avoid having to O(n) with the if